templates/jaasai/community.html

```diff
@@ -130,8 +130,8 @@ <h5>Discourse</h5>
       <p>Join the Juju community on our <a href="{{ external_urls.discourse }}">Discourse forum</a>.</p>
     </div>
     <div class="col-4">
-      <h5>Ask Ubuntu</h5>
-      <p>Find out more, get help or ask any question at <a class="p-link--external" href="{{ external_urls.askubuntu }}">Ask Ubuntu</a>.</p>
+      <h5>Discourse</h5>
+      <p>Find out more, get help or ask any question on our <a class="p-link--external" href="{{ external_urls.discourse }}">Discourse</a>.</p>
```

Comment:
It's a little weird that we have 2 items in the 3col that point to the same place, maybe @spencerbygraves can offer up some alternatives as a follow-up.

Reply:
Oops this comment was supposed to be in the other file :D

Reply:
Good point - is there anywhere else we can link to here @clagom ?

<img width="1219" alt="Screenshot 2019-07-30 at 10 07 49" src="https://user-images.githubusercontent.com/505570/62116361-f4f28200-b2b1-11e9-86c0-c6e9acd5ac6e.png">

Reply:
We usually have also GitHub and/or irc

Reply:
Ok, will swap out for IRC link.